data.sql

```diff
@@ -3,4 +3,5 @@
 INSERT INTO animals(id,name,date_of_birth, escape_attempts, neutered, weight_kg) VALUES(1
 ,'Agumon','2020-2-3',0,TRUE,10.23);
 INSERT INTO animals(id,name,date_of_birth, escape_attempts, neutered, weight_kg) VALUES(2,'Gabumon','2018-11-15',2,TRUE,8),(3,'Pikachu','2021-1-7',1,FALSE,15.04),(4,'Devimon','2017-5-12',5,TRUE,11);
+ INSERT INTO animals (id,name,date_of_birth, escape_attempts, neutered, weight_kg) VALUES(5,'Charmander','2020-02-08',0,FALSE,11),(6,'Plantmon','2021-11-15',2,TRUE,5.7),(7,'Squirtle','1993-04-02',3,FALSE,12.13),(8,'Angemon','2005-06-12',1,TRUE,45),(9,'Boarmon','2005-06-07',7,TRUE,20.4),(10,'Blossom','1998-10-13',3,TRUE,17),(11,'Ditto','2022-05-14',4,TRUE,22);
```

Comment:
- [ Required ] I think it would be nice if the data being inserted into the `animals` table is the same as that in the project requirements. In the project requirements, the first four entries have negative values for the `weight` column e.g `-11.0`, `-5.7`, `-12.13` and `-45.0` yet all your entries are positive in value. Please see the screenshot below for more details on this and rectify.

## Project requirements
![Screenshot from 2023-09-01 00-38-18](https://github.com/Chandan-devs-tech/Vet-clinic-database/assets/16392046/839516a7-14de-44d3-bb45-00f357549b5b)